### ISSUE TITLE
Question about error "mmco: unref short failure" in finetune stage.

### ISSUE THREAD

**Issue by L4zyy** [dvlab-research/LLaMA-VID#19]
Thanks for tharing this great work.

When I finetuning the model from the pretrained weights, I keep getting the error message:

`[h264 @ 0x7d53ee00] mmco: unref short failure`

Despite the error message, the training continued and the loss kept dropping.

After searching through the web, I found that this error may introduced by "decord" (https://github.com/dmlc/decord/issues/177) . I wonder if you encountered this problem before?

**Comment by L4zyy** [comment]
Got it. Thanks for the quick reply! 😃
